cargo-miri/bin.rs

```diff
@@ -646,6 +708,18 @@ fn phase_cargo_rustc(args: env::Args) {
     }
 }
 
+fn try_forward_patched_extern_arg(args: &mut impl Iterator<Item = String>, cmd: &mut Command) {
```

Comment:
Why "try"?

Reply:
Because it can panic on a malformed `--extern` argument, was my reasoning.

Reply:
`try` usually indicates an `Option` or `Result` return type... so I think it'd be better to remove it here.

Reply:
Done.